src/i18n/locale-en.json

```diff
@@ -2506,7 +2506,7 @@
     "common.show_all": "Show all",
     "common.hide": "Hide",
     "common.learn_more_in_documentation": "Learn more in the GraphDB documentation",
-    "common.copied": "Copied",
+    "common.copied": "IRI copied",
```

Comment:
Create a more specific key for the copy directive, e.g. `copy_to_clipboard.copied.tooltip`.